packages/runner-ct/src/app/RunnerCt.scss

```diff
@@ -28,7 +28,15 @@ body, html {
 }
 
 .runner {
-  @include checkerboard();
+  // @include checkerboard();
+   background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI1IiBoZWlnaHQ9IjUiPgo8cmVjdCB3aWR0aD0iNSIgaGVpZ2h0PSI1IiBmaWxsPSIjZGJkYmRiIj48L3JlY3Q+CjxwYXRoIGQ9Ik0wIDVMNSAwWk02IDRMNCA2Wk0tMSAxTDEgLTFaIiBzdHJva2U9IiNjNGM0YzQiIHN0cm9rZS13aWR0aD0iMSI+PC9wYXRoPgo8L3N2Zz4=");
```

Comment:
No nice CSS definition of the background?

Reply:
This is the existing background SVG of Cypress, just dupe'd with a lighter color. Yes, CSS-only is ideal, but the story wasn't to do that.